ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-gcp-ipi-ovn-sdn2ovn-provision-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-ovn-sdn2ovn-provision
+  steps:
+  - chain: ipi-conf-gcp
+  - chain: ipi-install
```

Comment:
I think we should adding `- ref: enable-qe-catalogsource` before ovn-sdn-migration.